apps/src/p5lab/animationListModule.js

```diff
@@ -281,22 +285,32 @@ export function setInitialAnimationList(serializedAnimationList) {
   }
 
   // TODO (from 2020): Tear out this migration when it hasn't been used for at least 3 consecutive non-summer months.
-  serializedAnimationList.orderedKeys.forEach(loadedKey => {
-    let animation = serializedAnimationList.propsByKey[loadedKey];
-    if (animation.sourceUrl && animation.sourceUrl.includes('/v3/')) {
-      // We want to replace this sprite with the /v1/ sprite
-      let details = `name=${animation.name};key=${loadedKey}`;
-      if (defaultSprites.propsByKey[loadedKey]) {
-        // The key is the same in the main.json and in default sprites. Do a simple replacement.
-        serializedAnimationList.propsByKey[loadedKey] =
-          defaultSprites.propsByKey[loadedKey];
-        trackEvent('Research', 'ReplacedSpriteByKey', details);
-      } else {
-        // We were unable to find a replacement for the /v3/ sprite
-        trackEvent('Research', 'CouldNotReplaceSprite', details);
+  if (shouldRunV3Migration) {
+    serializedAnimationList.orderedKeys.forEach(loadedKey => {
+      let animation = serializedAnimationList.propsByKey[loadedKey];
+      if (
+        !animation.sourceUrl ||
+        animation.sourceUrl.includes(dashboard.project.getCurrentId())
+      ) {
+        // The animation was created by the user. Skip.
```

Comment:
nit: s/user/project owner? Since the user viewing/playing the app isn't necessarily the project owner

Reply:
haha I had to think about that syntax for a moment. Great point. I'll update it :)